test/bugs/ui-4175.js

```diff
@@ -0,0 +1,68 @@
+// https://github.com/swagger-api/swagger-ui/issues/4175
```

Comment:
We broke away from this pattern `/test/bugs/*.js` some time ago. Can you please incorporate tests into https://github.com/swagger-api/swagger-js/blob/master/test/specmap/all-of.js? 

Second note, can you please use minimal OpenAPI definition as a data for your tests? That way the test will verify that it can process the OpenAPI definitions as we expect. THanks!

Reply:
Thanks for your review! Now the test also verifies definitions, not only examples.